app/controllers/introduction_introduce_yourself_controller.rb

```diff
@@ -3,15 +3,15 @@
 class IntroductionIntroduceYourselfController < StepsController
```

Comment:
Yeah I think the reason this is named this way is that `Introduction` is kind of a namespace. In the steps architecture. This makes more sense when you see the former `StepNavigation` code: https://github.com/codeforamerica/michigan-benefits/pull/22/files#diff-36e9c654f58b00a97974aba78e06ff24

For now, I am cool with keeping as-is since this PR is pretty large, but I do think we will want to do some renaming.